Getting started in new docs hub (#2231)
giantswarm/docs: src/content/getting-started/prepare-your-provider-account/azure/_index.md

    * An account with [Owner](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#owner) or [User Access Administrator](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#user-access-administrator) role.
    * [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) installed and configured to point to chosen subscription via [`az account set`](https://learn.microsoft.com/en-us/cli/azure/account?view=azure-cli-latest#az-account-set) command.

## Step 1: Service quotas {#quotas}

> Azure enforces [service quota through all cloud services](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits). The following overview lists the usual resources you may adjust depending on number of clusters and their size planned to be deployed:

Reviewer: ```suggestion
Azure enforces [service quota through all cloud services](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits). The following overview lists the usual resources you may adjust, depending on number and size of clusters planned to be deployed:
```

You can follow the request status in the **Quotas** section.

> ## Step 2:## Permissions

Author: ```suggestion
## Step 2: Permissions
```

    * Go to "Access control (IAM)"
    * Click "Add Role Assignment" and add the "Contributor" role and the "User Access Administrator" role to the App with the "subscription" Scope

#### Provide generated credentials to Giant Swarm

The following information needs to be provided to Giant Swarm:

> ```

Author: ```suggestion
```text
```

**Note**: This acceptance needs to be performed only once for a subscription that's used to run Giant Swarm workload clusters.

### Enable encryption at host

> If your security requirements demand this, there is a possibility to enable [encryption of data stored a virtual machine hosts](https://learn.microsoft.com/en-us/azure/virtual-machines/linux/disks-enable-host-based-encryption-cli). With this enabled, your data will be stored and transferred between storage and host encrypted, adding additional protecting from unauthorized access to your data.

Reviewer: ```suggestion
If your security requirements demand this, there is a possibility to enable [encryption of data stored on virtual machine hosts](https://learn.microsoft.com/en-us/azure/virtual-machines/linux/disks-enable-host-based-encryption-cli). With this enabled, your data will be stored and transferred between storage and host encrypted, adding additional protection from unauthorized access.
```
